Formula/zim.rb

```diff
@@ -0,0 +1,37 @@
+class Zim < Formula
+  desc "Graphical text editor used to maintain a collection of wiki pages"
+  homepage "http://zim-wiki.org/"
+  url "https://github.com/jaap-karssenberg/zim-desktop-wiki/archive/0.67.tar.gz"
+  sha256 "6817a4eb7be2326870810e4f4bc57c88128b2087752a8bd54953c95357b919fa"
+  head "https://github.com/jaap-karssenberg/zim-desktop-wiki.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+  depends_on "python"
+  depends_on "pygtk"
+  depends_on "pygobject"
+  depends_on "pygtksourceview" => :optional
+  depends_on "graphviz" => :optional
+
+  resource "pyxdg" do
+    url "https://files.pythonhosted.org/packages/source/p/pyxdg/pyxdg-0.25.tar.gz"
+    sha256 "81e883e0b9517d624e8b0499eb267b82a815c0b7146d5269f364988ae031279d"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec / "lib/python2.7/site-packages"
```

Comment:
The spacing around the `/` is unnecessary.

Reply:
Applies to everywhere else the spacing around `/` is present in this formula too.